sw/device/silicon_creator/lib/error.h

```diff
@@ -65,8 +65,9 @@ enum module_ {
   X(kErrorUartInvalidArgument,        ERROR_(1, kModuleUart, kInvalidArgument)), \
   X(kErrorUartBadBaudRate,            ERROR_(2, kModuleUart, kInvalidArgument)), \
   X(kErrorHmacInvalidArgument,        ERROR_(1, kModuleHmac, kInvalidArgument)), \
-  X(kErrorSigverifyBadEncodedMessage, ERROR_(1, kModuleSigverify, kInvalidArgument)), \
-  X(kErrorSigverifyBadExponent,       ERROR_(2, kModuleSigverify, kInvalidArgument)), \
+  X(kErrorSigverifyBadKey,            ERROR_(1, kModuleSigverify, kInvalidArgument)), \
+  X(kErrorSigverifyBadEncodedMessage, ERROR_(2, kModuleSigverify, kInvalidArgument)), \
```

Comment:
Can we add BadKey after the other errors to avoid changing the existing error codes? I think these error codes are meant to be append only, though I don't know if we need to strictly enforce that yet.

Reply:
Done.